assets/app/scripts/controllers/pods.js

```diff
@@ -20,8 +20,48 @@ angular.module('openshiftConsole')
     $scope.emptyMessage = "Loading...";
     var watches = [];
 
+    $scope.gotoContainerView = function(container, jolokiaUrl) {
+      // console.log("OPENSHIFT_CONFIG: ", OPENSHIFT_CONFIG);
+      // console.log("Clicked! container: ", container, " jolokiaUrl: ", jolokiaUrl);
+      var returnTo = window.location.href;
+      var title = Core.pathGet(container, ['name']) || 'Untitled Container';
+      var targetURI = new URI();
+      targetURI.path('/console/java/')
```

Comment:
all links should be made relative to the internal root... see https://github.com/openshift/origin/tree/master/assets#navigation